wire-ios-data-model/Tests/OneOnOne/OneOnOneMigratorTests.swift

```diff
@@ -212,7 +212,7 @@ final class OneOnOneMigratorTests: XCTestCase {
 
         // required to add be able to add images
         let cacheLocation = try XCTUnwrap(
-            FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first
+            FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first?.appendingPathComponent(UUID().uuidString)
```

Comment:
It could be nice to add an extension like

```swift
FileManager.default.randomCacheDirectoryURL()
```

Reply:
I'll do this change since @netbe is out